src/test/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/resource/ReportResourceTest.kt

```diff
@@ -1481,6 +1481,225 @@ class ReportResourceTest : SqsIntegrationTestBase() {
     }
   }
 
+  @DisplayName("POST /incident-reports/{id}/description-addendums")
+  @Nested
+  inner class AddDescriptionAddendum {
+    private lateinit var url: String
+
+    // language=json
+    private val validPayload = """
+      {
+        "createdAt": "2023-12-05T21:00:00",
```

Comment:
not convinced on the `createdAt` being in the request object as we are setting two different dates on the update

Reply:
no longer _required_ as of #329